testdata/scripts/goprivate.txt

```diff
@@ -21,8 +19,17 @@ go 1.15
 -- standalone/main.go --
 package main
 
-// TODO: This tests that #146 is fixed, but is blocked by errors garbling 'net'
-//import _ "net"
+import (
+	// This used to cause errors since the "net" import causes 'go list -json'
+	// to output ImportMap, since "net" imports packages vendored in std.
+	// TODO: This tests that #146 is fixed, but is blocked by errors garbling 'net'
+	// _ "net"
```

Comment:
Is this still needed to be commented out?

Reply:
Kind of. You're right that we should revisit this, though. The better std list, plus an extra three lines to fix another cgo edge case, mean that we can now obfuscate the net package :) The test now does check "net".